drivers/sensor/st/lsm6dsv16x/lsm6dsv16x_trigger.c

```diff
@@ -36,7 +36,7 @@ static int lsm6dsv16x_enable_xl_int(const struct device *dev, int enable)
 	}
 
 	/* set interrupt */
-	if (cfg->drdy_pin == 1) {
+	if ((cfg->drdy_pin == 1) || ON_I3C_BUS(cfg)) {
```

Comment:
Not very expert on I3C on our sensors, but I'm not finding anything on the technical docs that says that INT1 pin only can be used in I3C. Is this really the case? Can you point me to the docs?

Reply:
the int1 pin is automagically disabled (unless if you set a bit in a special reg). That int1 register for enabling the interrupts becomes the I3C IBI TIR enable register

Reply:
OK got it. And now I understand also that when I3C is involved there is no "GPIO int1 vd int2" thing, as interrupt will be received thru data lines.